kornia/geometry/transform/crop.py

```diff
@@ -13,6 +13,7 @@
     "bbox_to_mask",
     "infer_box_shape",
     "validate_bboxes",
+    "bbox_generator"
```

Comment:
@shijianjian I would start considering to have a dedicated submodule under `kornia.geometry.transform.boxes` with all the functionalities related to bounding boxes. At some point @lferraz also proposed this to me. /cc @ducha-aiki